sync/upstream.py

```diff
@@ -721,13 +721,16 @@ def status_changed(git_gecko, git_wpt, sync, context, status, url, sha):
     if status == "pending":
         # Never change anything for pending
         return landed
-
-    if status == "success":
+    previous_check = sync.last_pr_check
+    check_state, _ = env.gh_wpt.get_combined_status(sync.pr, exclude={"upstream/gecko"})
+    sync.last_pr_check = {"state": check_state, "sha": sha}
+    if check_state == "success":
         sync.error = None
         if sync.gecko_landed():
             landed = sync.try_land_pr()
-        else:
-            env.bz.comment(sync.bug, "Upstream web-platform-tests status checked passed, "
+        elif previous_check != sync.last_pr_check:
+            env.bz.comment(sync.bug,
+                           "Upstream web-platform-tests status checks passed, "
                            "PR will merge once commit reaches central.")
     else:
         env.bz.comment(sync.bug, "Upstream web-platform-tests status %s for %s. "
```

Comment:
This comment should only happen if the `previous_check != sync.last_pr_check` condition holds, correct? Should we just add the message here about which tests fail rather than in `try_land_pr` above (and not add anything to the bug in that case)?